puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -1,22 +1,38 @@
 # @api private
 class slave::postgresql {
-  Postgresql_psql {
-    cwd => '/',
-  }
+  # only CentOS slaves are used to run unit tests
+  if $::osfamily == 'RedHat' {
+    if ::operatingsystemmajrelease == '7' {
+      include scl
 
-  include ::postgresql::client
-  include ::postgresql::server
+      scl::collection { 'rh-postgresql10':
+        enable => true,
+      }
 
-  # Simple known user/pass that will allow Jenkins to create its
-  # own databases when required
-  postgresql::server::role { 'foreman':
-    password_hash => postgresql_password('foreman', 'foreman'),
-    superuser     => true,
-    login         => true,
-  }
+      $non_scl_rpms = ['postgresql', 'postgresql-server', 'postgresql-devel', 'postgresql-libs']
+      $non_scl_rpms.each |String $non_scl_rpm| {
+        package { "non_scl_${non_scl_rpm}":
+          name   => $non_scl_rpm,
+          ensure => 'absent'
+        }
+      }
+
+      class { 'postgresql::globals':
+        version              => '10',
+        client_package_name  => 'rh-postgresql10-postgresql-syspaths',
+        server_package_name  => 'rh-postgresql10-postgresql-server-syspaths',
+        contrib_package_name => 'rh-postgresql10-postgresql-contrib-syspaths',
+        service_name         => 'postgresql',
+        datadir              => '/var/opt/rh/rh-postgresql10/lib/pgsql/data',
+        confdir              => '/var/opt/rh/rh-postgresql10/lib/pgsql/data',
+        bindir               => '/usr/bin',
+      }
+      file { '/usr/bin/pg_config':
```

Comment:
I think that if you install `rh-postgresql10-syspaths` this shouldn't be needed.

Reply:
Nope...
```
[root@localhost ~]# rpm -q rh-postgresql10-syspaths
rh-postgresql10-syspaths-3.1-1.el7.x86_64
[root@localhost ~]# which pg_config
/usr/bin/which: no pg_config in (/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin:/usr/local/rvm/bin)

```